python/py-MDAnalysis/Portfile

```diff
@@ -25,28 +25,24 @@ master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  6ac6c5ad4ad83ff1227ec596cf47de76587d4478 \
-                    sha256  f45a024aca45e390ff1c45ca90beb2180b78881be377e2a1aa9cd6c109bcfa81 \
-                    size    19552874
-
-patchfiles          patch-unistd.diff
-
-if {${name} ne ${subport}} {
```

Comment:
this `if {${name} ne ${subport}} {` needs to stay

Reply:
I squash my commits and restored if {${name} ne ${subport}} {}